Sources/Variable.swift

```diff
@@ -74,7 +74,11 @@ public struct Variable : Equatable, Resolvable {
       if let context = current as? Context {
         current = context[bit]
       } else if let dictionary = current as? [String: Any] {
-        current = dictionary[bit]
+        if bit == "count" && dictionary[bit] == nil {
```

Comment:
Yes, there could potentially be a conflict there. 
What do you suggest the fix is? Always read `count` as the count of the dictionary? Rename it to a less likely name like `dictionaryCount`, but then lose consistency with array?